website/sales/api/v2/urls.py

```diff
@@ -10,6 +11,7 @@
 app_name = "sales"
 
 urlpatterns = [
+    path("sales/order/<uuid:pk>/pay/", OrderPaymentView.as_view(), name="order-pay"),
```

Comment:
I wouldn't call it `pay`, but really call it `claim`. Paying should happen via the payments app.

Reply:
Fair enough, also kind of a misnomer for the non-api version though, but I'll change it here

Reply:
The non-API view redirects to the payment view so it works a bit differently

Reply:
Let's not forget we should register the non api URL in the apple app site association file thingy so the app will open up

Reply:
True, but I'll do that only once the app also actually supports it